examples/with-typescript/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "with-typescript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "nuxt-ts",
+    "build": "nuxt-ts build",
+    "generate": "nuxt-ts generate",
+    "start": "nuxt-ts start"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
devDependencies need to be upgraded
![image](https://user-images.githubusercontent.com/25272043/72249924-bb60c980-35fa-11ea-92a9-97fdcdf9cd5d.png)

Reply:
Should be fixed by a rebase